app/core/model/src/main/java/io/syndesis/model/integration/Integration.java

```diff
@@ -117,8 +116,6 @@ default Kind getKind() {
 
     Optional<Date> getCreatedDate();
 
-    Optional<BigInteger> getTimesUsed();
```

Comment:
Shouldn't this go to an IntegrationRevision (not sure whether the corresponding PR has been already merged) ? Why does it need to be a BigInteger ? Isn't a long good enough ?

Reply:
Wouldn't just a `count` be better than `timesUsed` ?

Reply:
This is being removed so sholdn't be a problem. Are the @syndesisio/ui-api folks aware of this change?

Reply:
Ah yes. Guess I'm red/green blind ;-)